app/src/main/java/org/simple/clinic/util/AppSignature.kt

```diff
@@ -0,0 +1,51 @@
+package org.simple.clinic.util
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.pm.PackageManager
+import android.util.Base64
+import java.nio.charset.StandardCharsets
+import java.security.MessageDigest
+import java.util.Arrays
+
+private const val HASH_TYPE = "SHA-256"
+private const val NUM_HASHED_BYTES = 9
+private const val NUM_BASE64_CHAR = 11
+
+@SuppressLint("PackageManagerGetSignatures")
+/**
+ * This is a helper class to generate your message hash to be included in the SMS message.
+ *
+ * Without the correct hash, the app won't receive the message callback. This only needs to be
+ * generated once per app and stored.
+ */
```

Comment:
Can you also change the voice of this comment? Maybe also remove "helper class"?